### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HustlinTom**
I attempted to make the README.md a little more user friendly and thorough in the setup process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,71 +72,122 @@ Features:
...
+If you get an error 'groupadd: group 'docker' already exists', proceed
+
+    sudo usermod -aG docker $YOURUSERHERE$ 
+
+The two above commands allow for Docker commands to be executed without needing to call for sudo each time you run any Docker command.  Logout, then login again.
+
+    docker run --rm -it --init alpine echo "success"   
     
-Where the new download path would be `/home/bob/Downloads`.  Leave the right side alone, i.e `/downloads` as that maps to an internal container path.
+This ensures Tini, a zombie docker process killer, is activated
```
Comment (lardbit) [inline README.md]: I don't see any other reference to `Tini`.  Did you mean to include this?

File: README.md [hunk 2]
```diff
@@ -72,71 +72,122 @@ Features:
...
+
+### Part 1 - Setup performed from terminal
+
+    sudo systemctl start docker.service
+    sudo systemctl enable docker.service
+    sudo groupadd docker
+   
+If you get an error 'groupadd: group 'docker' already exists', proceed
+
+    sudo usermod -aG docker $YOURUSERHERE$
```
Comment (lardbit) [inline README.md]: `$USER` is the standard shell variable for the current user.  Using that instead, the user can just copy/paste this line in.

For instance, see your $USER:

    echo $USER

File: README.md (hunk 1 above)
Comment (HustlinTom) [reply README.md]: I did, and it was more for the benefit of those curious as to what each particular executed command accomplished.  If it seems unnecessary I'd be happy to remove it.

File: README.md (hunk 2 above)
Comment (HustlinTom) [reply README.md]: Good point.  I was thinking as a technical writer, not in usage of terminology.  I'll revise it.

**Comment by lardbit** [comment]
Great.  I'm merging your updates into a new branch.  I'll clean up some of the markdown and merge into master.  Thanks

**Comment by lardbit** [comment]
@HustlinTom, The **Part 1** in your instructions refers to `systemctl` commands.  Is this relevant for all the distributions you documented? I thought that was only included in certain distributions?

For example, you documented to run:
    sudo systemctl start docker.service
    sudo systemctl enable docker.service

**Comment by HustlinTom** [comment]
> @HustlinTom, The **Part 1** in your instructions refers to `systemctl` commands. Is this relevant for all the distributions you documented? I thought that was only included in certain distributions?
> 
> For example, you documented to run:
> 
> ```
> sudo systemctl start docker.service
> sudo systemctl enable docker.service
> ```

It definitely will for any distro that uses systemd, which I'm positive most of the distros of the past three years have capability for if it's not installed by default.

**Comment by lardbit** [comment]
Ok, got it.  Thanks.  `systemd` does seem to be a standard these days.  Yeah, Ubuntu 14.04 is so old now I don't know if I want to be including special instructions for such outdated versions.

**Comment by lardbit** [comment]
Thanks @HustlinTom!  The documentation is much more user friendly now.  Everything is merged into master.

**Pull request merged**